app.js

```diff
@@ -1,91 +1,121 @@
 
 //Greeting
-var userName = prompt ('What\'s your name?');
-// if(userName == null || userName === ''){
-//   userName = 'guest';
-// }
-if (!userName) userName = 'guest';
-alert('Hello ' + userName + '. Thanks for coming to my site!');
+function greeting() {
+  var userName = prompt ('What\'s your name?');
+  // if(userName == null || userName === ''){
+  //   userName = 'guest';
+  // }
+  if (!userName) userName = 'guest';
+  alert('Hello ' + userName + '. Thanks for coming to my site!');
+}
+greeting();
 
 //Setup //8 questions
+// Put all questions inside of an array.
 var questionArray = ['Did I attend UC Berkeley?', 'Is fullstack development part of my long term goals?', 'Do I currently work as a SDET?', 'Is Paris my dream vacation destination?', 'Do I have a dog named, Eli?', 'What is one of my favorite dishes?', 'Guess a number between 1 to 100', 'What are my favorite food destination cities?'];
-// var myAnswerArray = ['yes', 'yes', 'yes', 'yes'];
+
 //# of answers: 9
-var inputAnswerArray = ['yes', 'y', 'beef noodle soup', 'prime rib', '88', 'taipei', 'tokyo', 'san sebastian', 'paris'];
+// put all correct answers inside of an array.
+var inputAnswerArray = ['yes', 'y', 'beef noodle soup', 'prime rib', 88, 'taipei', 'tokyo', 'san sebastian', 'paris'];
+// Created variables to count correct and incorrect answers.
 var correctCounter = 0;
 var incorrectCounter = 0;
 //function to execute for loop
+// Set up loop to loop through all the questions in questionArray.
 function guessingGameLoop(){
   for(var i = 0; i < questionArray.length; i++){
     var answer = prompt(questionArray[i]);
-
     //verifying typeof answer variable
-    if(typeof answer === "string"){
-      var inputAnswer = answer.toLowerCase();
-      console.log('lowercased');
+        // This if else statement runs whenever user provides user input.
+    if(answer % 1 == 0) {
+      var inputAnswer = parseInt(answer);
+      console.log('converted to int');
     }else{
-      var inputAnswer = answer;
-      console.log('nothing done');
+      // convert user input to lower case.
+      var inputAnswer = answer.toLowerCase();
+      console.log('lower case');
     }
     if((i === 7) && inputAnswerArray.indexOf(inputAnswer) >= 5){
       console.log('That is correct! ' + inputAnswerArray[5] + ', ' + inputAnswerArray[6] + ', ' + inputAnswerArray[7] + ', and ' + inputAnswerArray[8] + ' are the answers');
```

Comment:
Since you used such a long and difficult to read string concatenation here with the indices of inputAnswerArray both here and on line 40, it would've been a good idea to store this strong in a variable up above and simply logged that here. This mucks up your if block a bit and hurts readability.